CHANGELOG.md

```diff
@@ -0,0 +1,16 @@
+NativeScript CLI Changelog
+================
+
+0.9.2 (2015, March 17)
+===
+
+### New
+
+* [Implemented #305](https://github.com/NativeScript/nativescript-cli/issues/305), [#322](https://github.com/NativeScript/nativescript-cli/issues/322): You can quickly add or update your platform runtime to a specific version by running `tns platform update platform@version`<br/>For example: `tns platform update ios@0.9.2-beta`
+* [Implemented #302](https://github.com/NativeScript/nativescript-cli/issues/302): You can configure proxy settings for the NativeScript CLI.
+
+### Fixed
+
+* [Fixed #312](https://github.com/NativeScript/nativescript-cli/issues/312): `tns platform add ios` does not preserve your app ID, if not default.
```

Comment:
ping @Fatme  - have we fixed this one for 0.9.2 or we've postponed it for 0.10 ?